src/sgame/sg_bot_nav.cpp

```diff
@@ -592,11 +592,24 @@ static bool BotShouldJump( gentity_t *self, const gentity_t *blocker, const glm:
 		&& trace.surfaceFlags & SURF_LADDER
 		;
 
+	if ( blocker->s.modelindex == BA_A_BARRICADE )
```

Comment:
I think we should check for ET_BUILDABLE since `modelindex` could have a different meaning for other entities.

Also this block ought to be skipped for humans since the normal trace already works correctly for them.

Reply:
done